docs/site/Relation-generator.md

```diff
@@ -0,0 +1,114 @@
+---
+lang: en
+title: 'Relation generator'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Relation-generator.html
+summary: summary
+---
+
+# Relation generator
+
+Page Contents
```

Comment:
A table of contents will be generated automatically.

Reply:
Page contents was removed from the Relation-generator.md.